src/test/java/bio/terra/integration/FileTest.java

```diff
@@ -120,13 +120,13 @@ public void fileUnauthorizedPermissionsTest() throws Exception {
         String filePath = "/foo/bar";
 
         DataRepoResponse<JobModel> launchResp = dataRepoFixtures.ingestFileLaunch(
-            custodian(), datasetId, profileId, gsPath + "/files/File%20Design%20Notes.pdf", filePath);
+            custodian(), datasetId, profileId, gsPath + "/files/File Design Notes.pdf", filePath);
```

Comment:
This is a change in existing behavior. I would advocate that it's a good thing, because it requires less thinking on the user's part and aligns with gsutil:
```
$ gsutil stat 'gs://jade-testdata/files/File%20Design%20Notes.pdf'
No URLs matched: gs://jade-testdata/files/File%20Design%20Notes.pdf

$ gsutil stat 'gs://jade-testdata/files/File Design Notes.pdf'
    Creation time:          Tue, 16 Apr 2019 14:00:01 GMT
    Update time:            Tue, 16 Apr 2019 14:00:01 GMT
    Storage class:          REGIONAL
    Content-Length:         301354
    Content-Type:           application/pdf
    Hash (crc32c):          Y5gazg==
    Hash (md5):             6n/gNRj/0F/OwjR5YwMJiA==
    ETag:                   CIHAifvi1OECEAE=
    Generation:             1555423201222657
    Metageneration:         1
```
But worth calling out.

Reply:
Yes, it is a good thing. Thanks!